core/src/main/java/com/linecorp/armeria/client/retry/RetryingHttpClientBuilder.java

```diff
@@ -36,11 +39,22 @@
 
     private int contentPreviewLength = DEFAULT_CONTENT_PREVIEW_LENGTH;
```

Comment:
`DEFAULT_CONTENT_PREVIEW_LENGTH` is currently `0`, which is not the valid `contentPreviewLength`. Now that we have a separate strategy interface that has no content streaming, how about using `Integer.MAX_VALUE` as the default?